src/bin/deku_node.ml

```diff
@@ -240,7 +261,13 @@ let node =
     let doc = "Path to the folder containing the node configuration data." in
     let open Arg in
     required & pos 0 (some string) None & info [] ~doc ~docv in
-
+  let minimum_block_delay =
+    let docv = "minimum_block_delay" in
+    let doc =
+      "Determines the minimum time the node will wait before propagating a \
```

Comment:
Would you add to the documentation the unit , format and minimum value.
In this PR
       "Determines the minimum time in secondes in floating number with a minimal value of 0. The node will wait before propagating a \"
       
       or if change to int 
              "Determines the minimum time in milli-secondes in int number with a minimal value of 0. The node will wait before propagating a \"

Reply:
resolved in the latest commit.